streams/src/test/java/org/apache/kafka/streams/processor/internals/PartitionGroupTest.java

```diff
@@ -206,6 +212,8 @@ public void testTimeTracking() {
         verifyBuffered(0, 0, 0);
         assertEquals(0.0, metrics.metric(lastLatenessValue).metricValue());
 
+        group.setPartitionTime(partition2, 100);
+        assertEquals(group.partitionTimestamp(partition2), 100);
```

Comment:
In addition, we should check if "stream time" was updated correctly.

nit: `assertEquals` take "excepted value" as first parameter, so you need to flip both (otherwise the error message would be confusing if the test fails)